### ISSUE TITLE
Houdini effects rollback 

### ISSUE THREAD

**Issue by pandaDS3** [iabem97/houdini-public#6]
I cant remove the effects, like android emoji, icon shape and custom boot logo. for unknown reasons Apple applications were not installed after the reset. I know that I can restore all these effects with iOS version update via dfu, but I'm waiting for jailbreak and I do not want to do this iPhone 5s, iOS 11.1.2

**Comment by pandaDS3** [comment]
P.S. after reset I found Cydia icon on home screen, but it inactive(open an close it instantly)

**Comment by bpluhar** [comment]
`Reset All Settings` in the Settings app. Do **NOT** select `Reset All Settings and Content` as that'll erase everything. Reset just the settings and your icon badges, CC, emojis and all will be restored to default.

**Comment by pandaDS3** [comment]
It didn’t help. I don’t care about android O emojis, custom boot load logo etc. I need working apple services as FaceTime, iMessage and find my friends. I have install FilzaJailed, how I can fix this with it?

**Comment by bpluhar** [comment]
So your iMessage, FaceTime and stuff isn't working? Can you send regular SMS? Can you download apps from the app store? You may have something in your hosts file that's blocking your connection to the Apple Servers for something like iMessage and FaceTime.

**Comment by pandaDS3** [comment]
i can download any apps from appstore, not apple apps (with cloud icon, that mean not-downloaded-apps)
![img_0018](https://user-images.githubusercontent.com/16509984/34777673-1f47bf64-f62c-11e7-9cd2-54d5077277db.PNG)
![img_0019](https://user-images.githubusercontent.com/16509984/34777675-1f90017a-f62c-11e7-8cf1-4ac0cc42ee83.PNG)
the problem appeared after applying some effects (icon shape, andriod O emoji and custom boot logo icon) in houdini

**Comment by vaheholtian** [comment]
Could you solve your problem? I am facing the same issue!

**Comment by pandaDS3** [comment]
No, restore to 11.2 in dfu or waiting for fully working jailbreak with cydia
